docs/jdbct4ref_guide/src/asciidoc/_chapters/introduction.adoc

```diff
@@ -0,0 +1,53 @@
+////
+/**
+ *@@@ START COPYRIGHT @@@
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * @@@ END COPYRIGHT @@@
+ */
+////
+
+[[introduction]]
+= Introduction
+
+This document describes how to use the {project-name} JDBC Type 4
+Driver. This driver provides Java applications running on a foreign
+platform with JDBC access to {project-name} SQL databases on the
+{project-name}.
+
+*Supported Java Releases:* The Type 4 driver requires Java enabled
+platforms that support JDK 1.4.1 or higher.
+
+
+[[type-4-driver-api-package]]
+== Type 4 Driver API Package
+
+The Type 4 driver package, `org.trafodion.t4jdbc`, is shipped with the
+driver software. For class and method descriptions, see the
+_{project-name} JDBC Type 4 Driver API Reference_.
+
+The {project-name} JDBC Type 4 Driver (hereafter, Type 4 driver)
+implements JDBC technology that conforms to the standard JDBC 3.0 Data
+Access API.
+
+To obtain detailed information on the standard JDBC API, download the
+JDBC API documentation: http://docs.oracle.com/en/java/.
```

Comment:
This is the latest link I can find to Java documentation. Please suggest an alternative.